packages/terra-form-select/src/Select.jsx

```diff
@@ -206,13 +206,14 @@ class Select extends React.Component {
     return (
       <Frame
         {...otherProps}
+        data-terra-select
         value={Util.value(this.props, this.state)}
         display={this.display()}
         onDeselect={this.handleDeselect}
         onSelect={this.handleSelect}
         placeholder={selectPlaceholder}
         dropdown={dropdownProps => (
-          <DropdownMenu intl={intl} {...dropdownProps}>
+          <DropdownMenu {...dropdownProps}>
```

Comment:
The menu is now wrapped in injectIntl and so it no longer needs to be passed down as a prop here.